manage_app.py

```diff
@@ -236,35 +333,70 @@ def check_service_up(url: str, service: str, discord=False):
     """
 
     print()
-    print(bcolors.YELLOW_IN +
-          "CHECKING SERVICE {}:{} ".format(ENV, service) + bcolors.CEND)
+    print(
+        bcolors.YELLOW_IN +
+        "CHECKING SERVICE {}:{} ".format(
+            os.getenv("DOCK_ENV"), service,
+        ) + bcolors.CEND,
+    )
 
     req_method = ""
 
     if service == "WEB":
```

Comment:
It's usually a good idea to try and make some code better when you work on it. Before adding the new services, we had two main services running and we introduced three more with this PR.

Therefore, code is really hard to debug and really repetitive. In fact, on the virtual machine the request to the `shortener` doesn't fail but for some reason fails the build.

What you need to at this point is to extract this code in a function:
Here's an example:

```python
def make_request(url, method_type):
    try:
        if method_type == "GET":
            response = requests.get(url, verify=ENV != "LOCAL")
        elif method_type == "POST":
            response = requests.post(url, verify=ENV != "LOCAL")
        else:
            return None, "Invalid method type"
    except Exception as e:
        return None, e
    
    return response, None
```

Then you can do the following:

```python
response, error = make_request(url, method_type)

if error is not None:
    handle_exception(msg, method_type, url, "WEB", error, discord)  # depends on which service you are checking
elif response.status_code != 200:
    handle_status_code_exception(msg, method_type, url, "WEB", response.status_code, discord)
```

You can further make the code more readable by setting up a dict with the service name as the key and it's url as value and iterate over it and call the functions required for validating whatever it's needed.

Reduced by probably ~100 lines of code.